app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -69,6 +70,8 @@ open class HomeActivity : AppCompatActivity() {
 
     private var sessionObserver: SessionManager.Observer? = null
 
+    private val hotStartMonitor = HotStartPerformanceMonitor()
```

Comment:
Do you think that it's a good idea to include this in every version of the application? If not, we could do something like Metrics.kt does and have a "production version" that is essentially a noop and instantiate that here when the build variant is not performancetest.

Reply:
What concerns do you have about it being in production versions?

Taking some guesses:
- Logging in production could be a problem but VERBOSE logging is not visible by default nor do I think this is particularly sensitive information (it can be mostly inferred from other Fenix logs anyway)
- Performance penalty could be a problem but we'll want to record this data for telemetry eventually anyway so I don't think it's avoidable

Reply:
> VERBOSE logging is not visible by default

I'll clean up the comment to clarify this.

Reply:
I was concerned mostly about performance. If we are planning to eventually record it with Glean (etc), then this is moot!